src/sql/workbench/browser/designer/designerIssuesTabPanelView.ts

```diff
@@ -114,6 +126,13 @@ class TableFilterListRenderer implements IListRenderer<DesignerIssue, DesignerIs
 				break;
 		}
 		templateData.issueIcon.className = `issue-icon ${iconClass}`;
+		if (element.moreInfoLink) {
```

Comment:
you need to add else block to clear the `templateData.issueMoreInfoLink.` the template will get reused sometimes.